import Foundation
import XCTest

class AboutErrorHandling: KoanTest{
    
    func test_CatchingError() {

    }

}
